go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -773,13 +776,25 @@ func (tsv *TabletServer) Begin(ctx context.Context, target *querypb.Target, opti
 		"Begin", "begin", nil,
 		target, options, true /* isBegin */, false, /* allowOnShutdown */
 		func(ctx context.Context, logStats *tabletenv.LogStats) error {
-			defer tabletenv.QueryStats.Record("BEGIN", time.Now())
+			startTime := time.Now()
```

Comment:
This is a slight behavior change -- previously the stats were incremented when the txThrottler rejects the operation, but now it's only counted if the begin actually occurs.

I think that's an improvement personally.